packages-exp/auth-exp/index.ts

```diff
@@ -47,64 +36,65 @@ export {
 // Core functionality shared by all clients
 export * from './src';
 
-// Additional DOM dependend functionality
+// Additional DOM dependend functionality; we need to import and then
+// export separately so that the rollup alias will work (for aliasing these
```

Comment:
interesting. Is it WAI, or a bug in the plugin? If it's a bug, it may be worth linking to the issue.

Reply:
It's WAI I think. It uses `resolveId` which is import statements [according to rollup](https://rollupjs.org/guide/en/#resolveid). See: https://github.com/rollup/rollup-plugin-alias/blob/master/src/index.js